app/models/event.rb

```diff
@@ -29,7 +29,17 @@ def notify_users
     EventNotifications.perform_async(id)
   end
 
-  def self.notify_users_available_todays
-    Event.find_by(start_date: DateTime.now).each(&:users_available)
+  def attend(user_id)
+    Attendance.create(event_id: id, user_id: user_id) unless existing?(user_id)
+  end
+
+  def cancel_attend(user_id)
```

Comment:
`cancel attendance` would be the correct way to say it, but why not just `cancel`?